builder/vmware/common/driver_fusion.go

```diff
@@ -17,20 +17,10 @@ import (
 	"github.com/hashicorp/packer-plugin-sdk/multistep"
 )
 
-const (
-	// VMware Fusion application name.
-	fusionProductName = "VMware Fusion"
-
-	// VMware Fusion versions.
-	// TODO: Update to best effort comply with the Broadcom Product Lifecycle.
-	minimumFusionVersion       = "6.0.0"
-	isoPathChangeFusionVersion = "13.0.0"
-)
-
 // Initialize version objects
 var (
-	minimumFusionVersionObj, _       = version.NewVersion(minimumFusionVersion)
-	isoPathChangeFusionVersionObj, _ = version.NewVersion(isoPathChangeFusionVersion)
+	fusionMinVersionObj, _           = version.NewVersion(fusionMinVersion)
```

Comment:
As mentioned, I think this would synergise well with `version.Must` here, so the error is not part of what is returned, but instead you have only the version if it suceeded to be parsed, or an immediate crash when the process launches, which will be caught during testing.

Reply:
I've made some updates but pending tests.